qa-test-app/src/main/java/com/mapbox/navigation/qa_test_app/view/MapboxNavigationViewCustomizedActivity.kt

```diff
@@ -60,11 +62,17 @@ class MapboxNavigationViewCustomizedActivity : AppCompatActivity() {
                 binding.navigationView.customizeViewBinders {
                     speedLimit = CustomSpeedLimitViewBinder()
                 }
+                binding.navigationView.customizeViewOptions {
+                    routeLineOptions = this@MapboxNavigationViewCustomizedActivity.routeLineOptions
```

Comment:
Can we add some different route line color and route line casing color, so that the difference is immediately visible. Right now the only difference is: one has vanishing route line enabled and other has it disabled. This is very subtle and easy to miss.